databricks/koalas/__init__.py

```diff
@@ -20,10 +20,30 @@
 from databricks.koalas.version import __version__  # noqa: F401
 
 
+def assert_python_version():
+    import logging
+
+    _major = 3
```

Comment:
I think it's fine to don't use leading understore because they are not exposed to the outer scope anyway.